java/ingest/ingest-core/src/test/java/sleeper/ingest/task/IngestTaskTest.java

```diff
@@ -227,20 +228,28 @@ void shouldSaveTaskWhenOnlyJobFails() throws Exception {
             // Given
             Queue<Instant> times = new LinkedList<>(List.of(
                     Instant.parse("2024-02-22T13:50:00Z"), // Start
-                    Instant.parse("2024-02-22T13:50:01Z"), // Job start + fail
-                    Instant.parse("2024-02-22T13:50:05Z"))); // Finish
+                    Instant.parse("2024-02-22T13:50:01Z"), // Job start
+                    Instant.parse("2024-02-22T13:50:05Z"), // Job finish
```

Comment:
Is it worth saying in the comment here that the job failed instead of finished? Same for the other tests in this class

Reply:
Done, thanks